compiler-rt/lib/asan/asan_flags.inc

```diff
@@ -116,6 +116,9 @@ ASAN_FLAG(bool, poison_partial, true,
           "stack buffers.")
 ASAN_FLAG(bool, poison_array_cookie, true,
           "Poison (or not) the array cookie after operator new[].")
+ASAN_FLAG(int, track_poison, 0,
```

Comment:
It does not say that it's used as count?

Reply:
rename to poison_history_size?

Reply:
Renamed and updated description